WordPressAuthenticator/Services/SignupService.swift

```diff
@@ -71,9 +71,16 @@ class SignupService {
                                     success(createdAccount, false, username, bearer_token)
         }, failure: { error in
             if let error = (error as NSError?) {
+                
+                var existingEmail = ""
+                if let errorData = error.userInfo[WordPressComRestApi.ErrorKeyErrorData] as? [String: String],
+                    let email = errorData.first?.value {
```

Comment:
I don't know if this is potentially confusing, as we're assuming that an error from the endpoint is an email address and that it's valid data to display to the user as an email address 🤔 

Perhaps a code comment could help here to explain? And I wonder if we should check the error code before using the data, so we can be sure it's the right kind of error?

Reply:
Hey @frosty .

I made the test a bit smarter. See what you think now. Thanks!